runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -564,7 +564,8 @@ func s:HandleEvaluate(msg)
     else
       let s:evalFromBalloonExprResult .= ' = ' . value
     endif
-    call s:OpenHoverPreview([s:evalFromBalloonExprResult], v:null)
+    let s:evalFromBalloonExprResult = split(s:evalFromBalloonExprResult, '\\n')
+    call s:OpenHoverPreview(s:evalFromBalloonExprResult, v:null)
```

Comment:
the screenshot in #10020 shows lines with `\n`, but here the pattern is `'\\n'` which would only match `\\n`.

Reply:
Well, it is really weird.
I've tested the following strings: "\n", "\\n", '\n' and '\\n' and only '\\n' worked.
My theory is that if you do `:echo "foo\\bar"` in vim, you obtain `foo\nbar` as a message.
So I suppose, it's really `\\n` that is in s:evalFromBalloonExprResult and not a litteral `\n`

Reply:
Actually it's my mistake, I didn't realize that `split()` takes a pattern, not a  literal.